From 70207c50703abf55dbe4e2b6f00a62d40daa5a74 Mon Sep 17 00:00:00 2001
From: Binyuan Lan <lby@rock-chips.com>
Date: Wed, 16 Dec 2020 09:31:55 +0000
Subject: [PATCH] arm64: configs: rockchip_defconfig: enable es7210

This is needed by rk3566-rk817-tablet-rkg11 with mic array

Signed-off-by: Binyuan Lan <lby@rock-chips.com>
Change-Id: I6c1083b402273bfa1492c9a2f851a76a5c71ab7d
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 41b1d2a27358..6dc981484ce5 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -649,6 +649,7 @@ CONFIG_SND_SOC_BT_SCO=y
 CONFIG_SND_SOC_CX2072X=y
 CONFIG_SND_SOC_DUMMY_CODEC=y
 CONFIG_SND_SOC_ES7202=y
+CONFIG_SND_SOC_ES7210=y
 CONFIG_SND_SOC_ES7243E=y
 CONFIG_SND_SOC_ES8316=y
 CONFIG_SND_SOC_ES8396=y
-- 
2.35.3

